Supress copyright check in tests which can differ based on current year.

  https://github.com/adobe-type-tools/afdko/issues/1589

Adapted from upstream:

  https://github.com/adobe-type-tools/afdko/commit/feebd77d9b6507a0b32f837535511be3c94d9c6f

diff --git a/tests/tx_test.py b/tests/tx_test.py
--- a/tests/tx_test.py
+++ b/tests/tx_test.py
@@ -1246,7 +1246,7 @@ def test_ufo_fontinfo_parsing(file, msg, ret_code):
     if (ret_code == 0):
         expected_path = generate_ps_dump(expected_path)
         output_path = generate_ps_dump(output_path)
-        assert differ([expected_path, output_path])
+        assert differ([expected_path, output_path, '-s'] + PFA_SKIP)
     else:
         arg = [TOOL, '-t1', '-f', ufo_input_path]
         assert subprocess.call(arg) == 6
